apps/mobile/src/features/balances/stacks/sip10-balance.tsx

```diff
@@ -83,6 +87,14 @@ function Sip10BalanceWrapper({ data, mode = 'full' }: Sip10BalanceWrapperProps)
                 contractId={item.asset.contractId}
                 quoteBalance={item.quote.totalBalance}
                 imageCanonicalUri={item.asset.imageCanonicalUri}
+                onPress={() => {
+                  // pass balance and quote balance to the sheet from here
```

Comment:
I am passing the balances to the sheet via a callback as we already have that information at this point. 

Without doing this we would need to call the balance hooks again in the sheet and then have 6 different setups for each scenario (excluding Runes) - `BTC`, `STX` and `SIP-10` both all balances and account balance.